src/test/java/edu/hawaii/its/api/controller/GroupingsRestControllerTest.java

```diff
@@ -669,36 +669,30 @@ public void updateDescriptionTest() throws Exception {
 
     @Test
     @WithMockUhUser
-    public void enableSyncDestTest() throws Exception {
-        String uri = REST_CONTROLLER_BASE + "groupings/" + GROUPING + "/syncDests/listserv/enable";
+    public void updateSyncDestTest() throws Exception {
+        String uri = REST_CONTROLLER_BASE + "groupings/" + GROUPING + "/syncDests/listserv/";
 
+        String enableUri = uri + "true";
         given(httpRequestService.makeApiRequest(eq(UID), anyString(), eq(HttpMethod.PUT)))
-                .willReturn(new ResponseEntity(HttpStatus.OK));
+                .willReturn(new ResponseEntity<>(HttpStatus.OK));
```

Comment:
Remove the `<>` from `new ResponseEntity` since any empty type declaration is not needed.

Reply:
Hmm, I would have thought that without it, we'd get Raw use warnings. But maybe not.

Reply:
That's right, I suggested removing it since the rest of the file does not use the <> either. It seems better with <>.